### PR TITLE
DOCS-1624: Reference a different manifest file for managed clusters

### PR DESCRIPTION
**Pull request by asincu**
Product Version(s):
Calico Enterprise and Calico Cloud

Issue:
https://tigera.atlassian.net/browse/DOCS-1624

Link to docs preview:

SME review:
- [x] An SME has approved this change. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [x] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Review by bmckercher123 — CHANGES_REQUESTED** [review]
Netlify build is failing because it cannot find `compliance-reporter-pod-managed.yaml`.

**Comment by asincu** [comment]
@bmckercher123 @ctauchen This file was recently added under the manifest folder in calico private. I think the docs team need to create an exception for it in order to have the build passing ?

**Comment by danudey** [comment]
> @bmckercher123 @ctauchen This file was recently added under the manifest folder in calico private. I think the docs team need to create an exception for it in order to have the build passing ?

I think this PR should be merged as either part of the standard docs PR for the relevant Calient release, or after the docs PR for that release is merged.

e.g. if this change affects Calico v3.17.1, then this change should be part of the docs PR for Calico v3.17.1, or we should merge this PR after v3.17.1 is released and the docs PR is merged.

**Comment by ctauchen** [comment]
@danudey 

Depends on when the fix is going out. But yes, if it's 3.17.1, we can't merge this until after the release.

We plan to introduce a new one-PR system for this kind of release in the future. But for now, we're just going to be holding on to them and publishing them in sequence after the publishing PR is merge. For patch releases, there shouldn't ever be more than a few. 

That said, this build is passing, which tells me that the new manifest is already being served as part of 3.17.0. Unless that was an error? If not, we can merge this right away.

**Comment by asincu** [comment]
@ctauchen I got some help from @rene-dekker that upload the new manifest to the S3 bucket and got the build passing.

File: calico-cloud/compliance/compliance-reports-cis.mdx
Comment (asincu) [inline calico-cloud/compliance/compliance-reports-cis.mdx:148]: @bmckercher123 Are these changes needed in this location or should I simply update `calico-cloud_versioned_docs/version-3.17/compliance/compliance-reports-cis.mdx` and `calico-cloud_versioned_docs/version-3.17/compliance/overview.mdx` ?

**Review by bmckercher123 — APPROVED** [review]
LGTM.

**Review by tmjd — APPROVED** [review]
LGTM

**Pull request merged**
